conda-recipe/bld.bat

```diff
@@ -1,11 +1,9 @@
-call "%ONEAPI_ROOT%\compiler\latest\env\vars.bat"
-if errorlevel 1 (
-    echo "oneAPI compiler activation failed"
-    exit /b 1
-)
+
+REM A workaround for activate-dpcpp.bat issue to be addressed in 2021.4
+set "LIB=%CONDA_PREFIX%\Library\lib;%CONDA_PREFIX%\compiler\lib;%LIB%"
 
 "%PYTHON%" setup.py clean --all
-"%PYTHON%" setup.py install
+"%PYTHON%" setup.py install --sycl-compiler-prefix=%CONDA_PREFIX%\Library
```

Comment:
Why is CONDA_PREFIX used here and in the file above? Shouldn't it be BUILD_PREFIX?

Reply:
```suggestion
"%PYTHON%" setup.py install --sycl-compiler-prefix=%BUILD_PREFIX%\Library
```

Reply:
@1e-to good eye :) yes this can remain "CONDA_PREFIX" (they are the same)